build_utils/fetch_assets.py

```diff
@@ -0,0 +1,259 @@
+  os.makedirs(f"{data_dir_base}/CE-Candidate-{base_version}/bin/clienttools", exist_ok=True)
+  os.makedirs(f"{data_dir_base}/CE-Candidate-{base_version}/bin/ide", exist_ok=True)
+  os.makedirs(f"{data_dir_base}/CE-Candidate-{base_version}/bin/plugins", exist_ok=True) 
+  os.makedirs(f"{data_dir_base}/CE-Candidate-{base_version}/docs", exist_ok=True)
+  # do CE platform
+  os.chdir(f"{data_dir_base}/CE-Candidate-{base_version}/bin/platform")
+  for distribution in redhat_distributions:
+    download_asset(f"{repo_url}/download/community_{version}/hpccsystems-platform-community_{version}{distribution}.x86_64_withsymbols.rpm")
+  for distribution in debian_distributions:
+    download_asset(f"{repo_url}/download/community_{version}/hpccsystems-platform-community_{version}{distribution}_amd64_withsymbols.deb")
+  # do CE clienttools
+  os.chdir(f"{data_dir_base}/CE-Candidate-{base_version}/bin/clienttools")
+  for distribution in redhat_distributions:
+    download_asset(f"{repo_url}/download/community_{version}/hpccsystems-clienttools-community_{version}{distribution}.x86_64_withsymbols.rpm")
+  for distribution in debian_distributions:
+    download_asset(f"{repo_url}/download/community_{version}/hpccsystems-clienttools-community_{version}{distribution}_amd64_withsymbols.deb")
+  download_asset(f"{repo_url}/download/community_{version}/hpccsystems-clienttools-community_{version}Darwin-x86_64.pkg")
+  download_asset(f"{repo_url}/download/community_{version}/hpccsystems-clienttools-community_{version}Windows-x86_64.exe")
+  # do CE plugins
+  os.chdir(f"{data_dir_base}/CE-Candidate-{base_version}/bin/plugins")
+  for plugin in plugin_list:
+    for distribution in redhat_distributions:
+      download_asset(f"{repo_url}/download/community_{version}/hpccsystems-plugin-{plugin}_{version}{distribution}.x86_64_withsymbols.rpm")
+    for distribution in debian_distributions:
+      download_asset(f"{repo_url}/download/community_{version}/hpccsystems-plugin-{plugin}_{version}{distribution}_amd64_withsymbols.deb")
+  # do CE eclide
+  os.chdir(f"{data_dir_base}/CE-Candidate-{base_version}/bin/ide")
+  download_asset(f"{repo_url}/download/community_{version}/hpccsystems-eclide-community_{version}Windows-i386.exe")
+  # do CE docs
+  for language in documentation_languages:
+    logging.info(f"Creating {data_dir_base}/CE-Candidate-{base_version}/docs/{language} directory")
+    os.makedirs(f"{data_dir_base}/CE-Candidate-{base_version}/docs/{language}", exist_ok=True)
+    os.chdir(f"{data_dir_base}/CE-Candidate-{base_version}/docs/{language}")
+    download_asset(f"{repo_url}/download/community_{version}/ALL_HPCC_DOCS_{language}-{version}.zip")
+    download_asset(f"{repo_url}/download/community_{version}/ECL_Code_Files_{language}.zip")
+    if os.path.exists(f"ALL_HPCC_DOCS_{language}-{version}.zip"):
+      with zipfile.ZipFile(f"ALL_HPCC_DOCS_{language}-{version}.zip", 'r') as zipped_docs:
```

Comment:
We now unzip the ALL_HPCC_DOCS file in the docs/language directory

Reply:
Looks good.